src/storage/conversion/RdfToQuadConverter.ts

```diff
@@ -2,18 +2,18 @@ import { PassThrough } from 'stream';
 import rdfParser from 'rdf-parse';
 import { BasicRepresentation } from '../../http/representation/BasicRepresentation';
 import type { Representation } from '../../http/representation/Representation';
+import type { ValuePreferences } from '../../http/representation/RepresentationPreferences';
 import { INTERNAL_QUADS } from '../../util/ContentTypes';
 import { BadRequestHttpError } from '../../util/errors/BadRequestHttpError';
 import { pipeSafely } from '../../util/StreamUtil';
+import { BaseTypedRepresentationConverter } from './BaseTypedRepresentationConverter';
 import type { RepresentationConverterArgs } from './RepresentationConverter';
-import { TypedRepresentationConverter } from './TypedRepresentationConverter';
 
-/**
- * Converts most major RDF serializations to `internal/quads`.
- */
-export class RdfToQuadConverter extends TypedRepresentationConverter {
+export class RdfToQuadConverter extends BaseTypedRepresentationConverter {
   public constructor() {
-    super(rdfParser.getContentTypesPrioritized(), INTERNAL_QUADS);
+    const inTypes = rdfParser.getContentTypes().then((types): ValuePreferences =>
+      Object.fromEntries(types.map((type): [string, 1] => [ type, 1 ])));
```

Comment:
Worth a utility function?

Reply:
As in the lost `toValuePreferences`?

Reply:
I forgot that function exists in the base typed converter. So this can just be replaced with `super(rdfParser.getContentTypes(), INTERNAL_QUADS);`